.github/workflows/nightly-pipeline.yaml

```diff
@@ -682,6 +682,25 @@ jobs:
       - ucxx-build
     if: ${{ !cancelled() }}
     runs-on: ubuntu-latest
+    steps:
+      - uses: rapidsai/trigger-workflow-and-wait@v1
+        with:
+          owner: rapidsai
+          repo: integration
+          github_token: ${{ secrets.WORKFLOW_TOKEN }}
+          github_user: GPUtester
+          workflow_file_name: build.yaml
+          ref: ${{ fromJSON(needs.get-run-info.outputs.obj).branch }}
+          wait_interval: 120
+          client_payload: ${{ toJSON(fromJSON(needs.get-run-info.outputs.obj).payloads.integration) }}
+          propagate_failure: true
+          trigger_workflow: true
+          wait_workflow: true
+  integration-test:
+    needs:
+      - integration-build
+    if: ${{ !cancelled() }}
```

Comment:
I do think it's fine to always run the `integration` test since it is really checking how out dated the solve is, but should add a comment explaining why this condition is different than all the others.

Reply:
I agree with this -- thanks!